<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FillMdWindow</class>
 <widget class="QDialog" name="FillMdWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>622</width>
    <height>340</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FillMD@Mks: fill genotyping errors with missing data</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="margin">
    <number>6</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="inputOutputVLayout">
     <item>
      <widget class="QLabel" name="IntroLabel">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <property name="text">
        <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Sans Serif'; font-size:9pt;&quot;&gt;Genotyping errors coming from the genotypes/pedigree file (.dat) file and recorded in the “events_summary.log” file (if present) can be filled with missing data (“-”) after that the consistency of marker genotyping information has been checked along generations of selection.&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'Sans Serif'; font-size:9pt;&quot;&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Sans Serif'; font-size:9pt;&quot;&gt;A new map file (.cmap) has been created to run OptiMAS marker by marker, cycle by cycle, in order to check genotyping errors along the pedigree. A new genotypes/pedigree file is created with genotyping errors filled with missing data.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
       <property name="margin">
        <number>3</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="inputGroupBox">
       <property name="title">
        <string>Data file to import:</string>
       </property>
       <layout class="QVBoxLayout" name="importBoxVLayout">
        <property name="margin">
         <number>6</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="cmapHLayout">
          <item>
           <widget class="QLabel" name="cmapLabel">
            <property name="minimumSize">
             <size>
              <width>154</width>
              <height>29</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>154</width>
              <height>29</height>
             </size>
            </property>
            <property name="toolTip">
             <string>e.g. events_summary.log</string>
            </property>
            <property name="text">
             <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Sans Serif'; font-size:9pt;&quot;&gt;Special map file (.cmap):&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="cmapLineEdit">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>29</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>29</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="cmapBrowseToolButton">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>29</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>29</height>
             </size>
            </property>
            <property name="text">
             <string>Browse...</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="cmapToolButton">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="configdialog.qrc">
              <normaloff>:/images/help.png</normaloff>:/images/help.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
            <property name="autoRaise">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="genoHLayout">
          <item>
           <widget class="QLabel" name="genoLabel">
            <property name="minimumSize">
             <size>
              <width>154</width>
              <height>29</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>154</width>
              <height>29</height>
             </size>
            </property>
            <property name="text">
             <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Sans Serif'; font-size:9pt;&quot;&gt;G&lt;/span&gt;&lt;span style=&quot; font-family:'Sans Serif'; font-size:9pt;&quot;&gt;e&lt;/span&gt;&lt;span style=&quot; font-family:'Sans Serif'; font-size:9pt;&quot;&gt;notypes/Pedigree file:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="genoLineEdit">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>29</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>29</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="genoBrowseToolButton">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>29</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>29</height>
             </size>
            </property>
            <property name="text">
             <string>Browse...</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="genoToolButton">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="configdialog.qrc">
              <normaloff>:/images/help.png</normaloff>:/images/help.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
            <property name="autoRaise">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="outputGroupBox">
       <property name="title">
        <string>Output directory:</string>
       </property>
       <layout class="QVBoxLayout" name="outputBoxVLayout">
        <property name="margin">
         <number>6</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="outputHLayout">
          <item>
           <widget class="QLabel" name="resultsLabel">
            <property name="minimumSize">
             <size>
              <width>154</width>
              <height>29</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>154</width>
              <height>29</height>
             </size>
            </property>
            <property name="toolTip">
             <string>Creation of the new genotype/pedigree file</string>
            </property>
            <property name="text">
             <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Sans Serif'; font-size:9pt;&quot;&gt;Results (new &amp;quot;.dat&amp;quot; file):&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="resultsLineEdit">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>29</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>29</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="resultsBrowseToolButton">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>29</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>29</height>
             </size>
            </property>
            <property name="text">
             <string>Browse...</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="resultsToolButton">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="configdialog.qrc">
              <normaloff>:/images/help.png</normaloff>:/images/help.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
            <property name="autoRaise">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="buttonsFrame">
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="spacing">
         <number>6</number>
        </property>
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="buttonsHLayout">
          <item>
           <spacer name="buttonsHBoxHSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="runPushButton">
            <property name="text">
             <string>Run</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="closePushButton">
            <property name="text">
             <string>Close</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="helpPushButton">
            <property name="text">
             <string>Help</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="configdialog.qrc"/>
 </resources>
 <connections/>
</ui>
